.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: java
 jdk:
   - oraclejdk8
+  - openjdk7
 # DISABLED openjdk7 as it still gives me a buffer overflow fault.
```

Comment:
Sorry, forgot to do that
The link below should also be changed to https://github.com/travis-ci/travis-ci/issues/5227#issuecomment-165131913